README.md

```diff
@@ -228,3 +504,45 @@ You can use any WP All Export Pro add-on by simply adding it's slug like so:
 For example:
 
 `junaidbhura/wpae-acf-add-on`
+
+### WPML Add-Ons
+
+You can use any WPML add-on by simply adding it's slug like so:
+
+`junaidbhura/<plugin-slug>`
+
+For example:
+
+`junaidbhura/wpml-sitepress-multilingual-cms`
+
+`junaidbhura/wpml-string-translation`
+
+`junaidbhura/wpml-media-translation`
+
+`junaidbhura/wpml-woocommerce-multilingual`
+
+`junaidbhura/wpml-gravityforms-multilingual`
+
+`junaidbhura/wpml-contact-form-7-multilingual`
+
+`junaidbhura/wpml-ninja-forms`
+
+`junaidbhura/wpml-wpforms`
+
+`junaidbhura/wpml-buddypress-multilingual`
+
+`junaidbhura/wpml-acfml`
+
+`junaidbhura/wpml-all-import`
+
+`junaidbhura/wpml-mailchimp-for-wp`
+
+`junaidbhura/wpml-wp-seo-multilingual`
+
+`junaidbhura/wpml-types`
+
+`junaidbhura/wpml-sticky-links`
+
+`junaidbhura/wpml-cms-nav`
+
+`junaidbhura/wpml-translation-management`
```

Comment:
This section is derived from from WP All Import/Export Pro and both should be rephrased to highlight the fact that only specific add-ons of each plugin are supported. The section should be phrase as so:

```suggestion
### WPML Add-Ons

The following WPML plugins and add-ons are supported:

* `junaidbhura/wpml-acfml`
* `junaidbhura/wpml-all-import`
* `junaidbhura/wpml-buddypress-multilingual`
* `junaidbhura/wpml-cms-nav`
* `junaidbhura/wpml-contact-form-7-multilingual`
* `junaidbhura/wpml-gravityforms-multilingual`
* `junaidbhura/wpml-mailchimp-for-wp`
* `junaidbhura/wpml-media-translation`
* `junaidbhura/wpml-ninja-forms`
* `junaidbhura/wpml-sitepress-multilingual-cms`
* `junaidbhura/wpml-sticky-links`
* `junaidbhura/wpml-string-translation`
* `junaidbhura/wpml-translation-management`
* `junaidbhura/wpml-types`
* `junaidbhura/wpml-woocommerce-multilingual`
* `junaidbhura/wpml-wp-seo-multilingual`
* `junaidbhura/wpml-wpforms`
```